test/e2e/test_cluster_configs/containerd.json

```diff
@@ -8,7 +8,7 @@
                 "kubernetesConfig": {
                     "networkPlugin": "azure",
                     "containerRuntime": "containerd",
-                    "windowsContainerdURL": "https://github.com/containerd/containerd/releases/download/v1.4.0-rc.0/containerd-1.4.0-rc.0-windows-amd64.tar.gz"
+                    "windowsContainerdURL": "https://github.com/containerd/containerd/releases/download/v1.4.0/containerd-1.4.0-windows-amd64.tar.gz"

```

Comment:
the same URL is also used in two examples under examples/windows, should those be changed as well?

Reply:
Yes - let me update

Reply:
Updated :)